src/components/RichTextEditor/Plugins/TextStylePlugin/TextStyleDropdown/DropdownItem.tsx

```diff
@@ -29,7 +31,7 @@ const isInList = (editor: PlateEditor) =>
 export const DropdownItem = ({ editor, type, children }: DropdownItemProps) => {
     const isActive =
         !!editor?.selection && isInList(editor)
-            ? getMark(editor, MARK_TEXT_STYLE) === type
+            ? getMark(editor, MARK_TEXT_STYLE) === type.id
```

Comment:
not sure if this is the line of code... Just noticed, that the dropdown item of the current textStyle is not active
![image](https://github.com/Frontify/fondue/assets/42832501/a7d826ac-bdfc-41f1-a583-4dd9e67faa24)

should be like this
![image](https://github.com/Frontify/fondue/assets/42832501/9715d5e4-fa8e-4d56-8865-4e01f3e87412)